lib/src/components/NxButton/__tests__/NxButton.test.tsx

```diff
@@ -65,4 +65,37 @@ describe('NxButton', function() {
     expect(() => render(<NxButton variant="icon-only" title="Save" disabled />))
         .toThrow(TypeError);
   });
+
+  it('applies small modifier class when provided in className', function() {
```

Comment:
Redundant with "it adds specified classNames to the element in addition to the defaults" test on line 24

Reply:
good point, removed